dvc/logger.py

```diff
@@ -58,97 +52,53 @@ class ColorFormatter(logging.Formatter):
     }
 
     def format(self, record):
+        info = record.msg
+
         if record.levelname == "INFO":
-            return record.msg
-
-        if record.levelname == "ERROR" or record.levelname == "CRITICAL":
-            exception, stack_trace = self._parse_exc(record.exc_info)
-
-            return (
-                "{color}{levelname}{nc}: {description}" "{stack_trace}\n"
-            ).format(
-                color=self.color_code.get(record.levelname, ""),
-                nc=colorama.Fore.RESET,
-                levelname=record.levelname,
-                description=self._description(record.msg, exception),
-                msg=record.msg,
-                stack_trace=stack_trace,
+            return info
+
+        if record.exc_info:
+            _, exception, _ = record.exc_info
+            cause = exception.__cause__
+            trace = "".join(traceback.format_exception(*record.exc_info))
```

Comment:
This should only be formatted under DEBUG if and should somehow include cause traceback. Ideally cause traceback should not be appended, but separated the same as Python does by default.